src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -65,6 +65,18 @@ internal static void Init()
                 LanguagePrimitives.GetEnumerator(null);
             });
         }
+
+        internal static IDictionary InitialEnvironmentVariables
+        {
+            get => _environmentVariables;
+        }
+
+        private static IDictionary _environmentVariables;
```

Comment:
Style: usually a field definition appears before it is used.

Reply:
Fixed.